com.unity.formats.fbx/Editor/FbxExporter.cs

```diff
@@ -483,7 +488,7 @@ private static bool ExportUVs(FbxMesh fbxMesh, MeshInfo mesh, int[] unmergedTria
         /// <summary>
         /// Export the mesh's blend shapes.
```

Comment:
Export the blend shapes of the mesh and connect them to the node? That would match the wording for ExportMesh.

I'm curious how this works with instancing.